src/constants.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+
+RECIDIVIZ_DATA_URL = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vTbxP67VHDHQt4xvpNmzbsXyT0pSh_b1Pn7aY5Ac089KKYnPDT6PpskMBMvhOX_PA08Zqkxt4zNn8_y/pub?gid=0&single=true&output=csv'
+MAPPINGS_URL = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vT95DUfwcHbauuuMScd1Jb9u3vLCdfCcieXrRthNowoSbrmeWF3ibv06LkfcDxl1Vd97S5aujvnHdZX/pub?gid=1112897899&single=true&output=csv'
+
+OUTPUT_START_DATE = datetime.date(2020, 3, 31)
+
+OUTPUT_COLUMNS = ['Date',
+                  'Facility Type',
+                  'State',
+                  #'County',
+                  'Canonical Facility Name',
+                  'Pop Tested',
+                  'Pop Tested Positive',
+                  'Pop Tested Negative',
+                  'Pop Deaths',
+                  'Staff Tested',
+                  'Staff Tested Positive',
+                  'Staff Tested Negative',
+                  'Staff Deaths',
+                  'Source',
+                  'Compilation',
+                  'Notes'
+                  ]
+
+SOURCE_FACILITY_NAME_COLUMN = 'Source Facility Name'
+
+COLUMN_MAPPINGS = {
+  'covidprisondata.com': {'State': 'State',
```

Comment:
Would be nice to have these column headers be an enum instead of raw strings (and case-sensitive ones at that). But if that's a larger change than you can perform right this second, let's file an issue for it.

Reply:
👍 will file an issue

Reply:
https://github.com/Recidiviz/covid19-aggregator/issues/3